packages/back-end/src/scripts/experiments.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "name": "checkout-layout",
+    "trackingKey": "checkout-layout",
+    "owner": "test@notarealemail.com",
```

Comment:
In the event we ever start sending out emails to owners, I think it's best we don't store emails that are under domains we don't own. Might be best to do something like `test-user@growthbook.io`.

Reply:
Oh, these files shouldn't have been picked up and are no longer needed in this PR, sorry.